evap/evaluation/fixtures/test_data.json

```diff
@@ -31,6 +31,7 @@
     "name_en": "Winter term 2012/2013",
     "participations_are_archived": true,
     "grade_documents_are_archived": false,
+    "results_are_archived": false,
```

Comment:
same, would be nice to have one semester with this set to true in the test data.